Buildings/Experimental/DHC/CentralPlants/Cooling/Controls/ChilledWaterBypass.mo

```diff
@@ -0,0 +1,96 @@
+within Buildings.Experimental.DHC.CentralPlants.Cooling.Controls;
+model ChilledWaterBypass
```

Comment:
General comment about this model:

1. The variable `numChi` suggest that it could be used for 1 or 2 chillers, but actually, it could not be used with 2 chillers without breaking the statement `connect(chiOn[2],mFloSet.u2)` which require that `chiOn` has at least two elements.
2. The switch is not necessary, since the output could be generalized to the product of `mMin_flow` and the number of chillers that are on. That value can be accessed by connecting `chiOn` to an array of `BooleanToInteger` followed by a `MultiSum`. Doing that would also make the block work for any number of chillers.
3. The PID setpoint should not be the on signal of the first chiller. My inclination is that the signal should not be normalized and the setpoint should directly be the aforementioned product `mMin_flow * numChiOn`. If you want to keep a normalized signal, then the setpoint should be a logic that check if ANY chiller is on, since that block could be used with chillers that are not staged in order.

Reply:
@bravache thank you for your comment. To generalize the controller, we implemented what you suggested above (see implementation figure below). We also took out the normalization block to avoid possible "division by zero" error when no chiller is on. Let us know your thoughts. Thanks! 

For reference, the normalized input for mass flow control was suggested by Michael earlier. My understanding is that it ensures the difference between the measurement and the setpoint is always within a certain scale. In this case, the gain `k` of the PI controller will work for different application cases. Let us know if we need to take out the normalized control in other control blocks as well. Thanks!

![image](https://user-images.githubusercontent.com/31628742/124788774-e897ea80-df06-11eb-89a2-0cb374367808.png)

Reply:
> Thanks for the changes. I have a better understanding of why the input signal was normalized, and I think that it would be useful to bring it back (sorry for the back and forth). @ChengnanShi-Work aptly pointed out that by normalizing the input signal, you don't need to scale the PID gain with the signal magnitude, and I think it is a good feature to have.
> Could you please add back to input signal normalization?

Sure, no problem. We have added back the normalization and also solved the “divided by zero” problem.